deploy/sre-prometheus/100-cluster-version-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-cluster-version-operator
+    role: alert-rules
+  name: sre-cluster-version-operator
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-cluster-version-operator
+    rules:
+    - alert: ClusterOperatorDegraded
```

Comment:
Will this alert name cause a collision? Can we have multiple alerts defined with the same alert name?

Generally, when SRE re-create an alert we use the pattern `AlertNameSRE` to make it clear it's not an upstream alert. I would suggest renaming it to `ClusterOperatorDegradedSRE` to align with the rest.